apps/entry/views.py

```diff
@@ -49,7 +52,41 @@
 import django_filters
 
 
-class EntryViewSet(viewsets.ModelViewSet):
+class EntrySummaryPaginationMixin(object):
+    def get_paginated_response(self, data):
+        if not self.request.data.get('calculate_summary',
+                                     self.request.GET.get('calculate_summary', '0')) == '1':
+            return super().get_paginated_response(data)
+        qs = self.filter_queryset(self.get_queryset())
+        q = qs.annotate(org=models.functions.Coalesce('lead__authors__parent', 'lead__authors')).values('org').annotate(org_type=models.functions.Coalesce('lead__authors__parent__organization_type', 'lead__authors__organization_type')).values('org_type').order_by('org_type').annotate(count=models.Count('org', distinct=True)).values('org_type', 'count')
```

Comment:
Very long. Please break this to multiple lines.
```suggestion
        q = qs.annotate(
                org=models.functions.Coalesce('lead__authors__parent', 'lead__authors')
            ).values('org').annotate(
                org_type=models.functions.Coalesce(
                    'lead__authors__parent__organization_type',
                    'lead__authors__organization_type'
                )
            ).values('org_type').order_by('org_type').annotate(
                count=models.Count('org', distinct=True)
            ).values('org_type', 'count')
```